change/@ni-nimble-components-752d1b02-6122-4d2f-b737-809aff6729d2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
One interesting behavior I noticed in initial Storybook testing: if you start a column resize and then release your mouse outside the browser, the drag operation isn't canceled. The result is that if you then bring your mouse back into the browser, the drag operation is still happening even though the mouse is no longer down. I'm seeing this on Safari and Edge on macOS but interestingly not Firefox (unless I'm doing something weird, I was testing in a rush).

Reply:
A similar weird behavior: start the drag with a right click instead of a left click. The browser context menu appears but if you dismiss it there is still a drag operation in progress even though the mouse isn't pressed.

Reply:
Regarding your first issue, I think this is a Storybook quirk. You don't see the same behavior in the Angular application. Additionally, you don't see the noted behavior in Chrome with vanilla JS apps in StackBlitz that have a similar setup. Not sure how we want to proceed given that.

For the second issue, I think I can just be more guarded in my event handling where I only process left clicks (or the "primary" button).

Reply:
For the first issue, I can confirm that it's only reproducible in Storybook and not in the Angular example app. That may mean it's reproducible in cases where clients use iframes which is not ideal but also likely not a priority for us to fix. I found a couple threads that seem related but didn't look in detail whether there are fixes we could leverage:
- [SO post suggesting it's related to `preventDefault`](https://stackoverflow.com/questions/66703382/how-to-keep-watching-mouse-move-outside-iframe)
- [Firefox bug suggesting the behavior in other browsers is correct](https://bugzilla.mozilla.org/show_bug.cgi?id=1681647)

I'm fine to leave as-is if those threads don't yield a simple fix.